### ISSUE TITLE
Question Help -> Can anyone share here a binaries or Simple application that uses OpencvSharp that uses Cuda

### ISSUE THREAD

**Issue by blaisexen** [shimat/opencvsharp#626]
@korhun
@arslanekhan
@9tontruck
@ ANYONE with good heart

Question Help -> Can anyone share here binaries or Simple application that uses OpencvSharp with  Cuda?

I just want the binaries of Opencvsharp with Cuda,  you can protect your Simple application if needed that you want.

a link would be nice.

I can share to you my CUDA application with Emgc_Cv,[Link Deleted Issue closed]
[here's the link](https://drive.google.com/open?id=10XxxBInIrW4KsW1nAUWRtYDaJkXUzG)

so, in short, I hope you understand what I meant, just like that, any SIMPLE application that uses Opencvsharp with CUDA would be nice "physical sample files" for others too.

thanks

**Comment by blaisexen** [comment]
@shimat 

hi,
Sorry for bothering you,

But I have something to tell, surprisingly I was able to run OpenCvSharp(3.4.5) with Cuda10

Now with OpenCv_4.0.1
thanks for this wonderful software.

**Comment by blaisexen** [comment]
Question closed.
I solved it,

**Comment by korhun** [comment]
Hi @blaisexen
(If I don't rememer it wrong) I had a try and was unsuccessful to use GPU. Can you tell how you have managed to solve this?

**Comment by frank-ozgul** [comment]
Could you share it with us?

**Comment by blaisexen** [comment]
hi to all,
here's the answer of all your questions,
https://github.com/shimat/opencvsharp/issues/627

I meant I solved it by understanding that @shimat  doesn't support OpenCvSharp with Gpu/Cuda.

**Comment by blaisexen** [comment]
@frank-ozgul 
@korhun 
and to All @

Sorry for the delay, here's what I got.
https://sourceforge.net/projects/opencvsharp-live-face-detect/

**Comment by blaisexen** [comment]
@frank-ozgul 
@korhun 
and to All @

Sorry for the delay, here's what I got.
https://sourceforge.net/projects/opencvsharp-live-face-detect/

**Comment by blaisexen** [comment]
@frank-ozgul 
@korhun 

any comment before I closed this question/issue?

**Comment by frank-ozgul** [comment]
@blaisexen 
Thank you for your efforts. I will try it in a proper time.

**Comment by korhun** [comment]
Hi @blaisexen
I have tried your code. NativeMethods.cuda_getCudaEnabledDeviceCount() value is still zero on my computer. My display adapter is NVidia GeForce GTX 1080 Ti.

**Comment by blaisexen** [comment]
@korhun 
No, do not try my code from your code or namespace, try my application from \BIN folder\0penCvSharp.Cuda.x86.FaceDetect.EXE

waiting for your feedback again, what's the result? "Application running CPU"? or "Application running CUDA"?
